### DISCUSSION TITLE
Screen Capture Implementation in Android

### DISCUSSION THREAD

**Discussion by JamshedAlamQaderi** [shepeliev/webrtc-kmp#101] [General]
Hi @shepeliev ,
At first thank you for this great library for kmp. After using it in multiple project, i want to contribute on screen capture functionality for android. So, my question is, do you have ongoing work for that or i could implement the screen capture functionality? 
Please let me know i want to help

**Comment by shepeliev** [comment]
Hi @JamshedAlamQaderi, thanks for your feedback. I appreciate it. At the moment no ongoing work for screen capturing so it would be cool if you contribute.

**Reply by JamshedAlamQaderi** [reply]
Thank you for the reply @shepeliev 
I'll start work on that
